packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_GDSWInterfacePartitionOfUnity_def.hpp

```diff
@@ -264,7 +264,7 @@ namespace FROSch {
             for (UN i=0; i<Vertices_->getNumEntities(); i++) {
                 for (UN j=0; j<Vertices_->getEntity(i)->getNumNodes(); j++) {
                     for (UN k=0; k<dofsPerNode; k++) {
-                        tmpVector->replaceLocalValue(Vertices_->getEntity(i)->getGammaDofID(j,k),i,1.0);
+                        tmpVector->replaceLocalValue(Vertices_->getEntity(i)->getGammaDofID(j,k),i,Teuchos::ScalarTraits<SC>::one());
```

Comment:
Consider using `withLocalAccess` here to access the Vector's local data structure.  See examples in Tpetra.